### PR TITLE
Add laser raptors

### PR DESCRIPTION
**Pull request by metalgearsloth**
https://github.com/space-wizards/space-station-14/assets/31366439/bd22aae1-6b6e-4cda-89d7-43d3533c6c33

:cl: Aleksh
- add: Added laser raptors.

### CODE REVIEW

**Comment by Ilya246** [comment]
it should play the weh sound when fired

**Comment by dmnct** [comment]
Why does it have petting-success-slimes ?

**Comment by Anzarot121** [comment]
We are Warriors
Wielding the Hammer so glorious
Fighting with the power of the Laser Dragon Fire

**Comment by AjexRose** [comment]
damn, leviathans wargames keep jumping to new extremes.

**Comment by ps3moira** [comment]
When petted it should play weh if successfully pet, maybe even a dinosaur scream and noises.

File: Resources/Prototypes/Entities/Mobs/NPCs/miscellaneous.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      fireRate: 1
+      useKey: false
+      selectedMode: SemiAuto
+      availableModes:
+        - SemiAuto
+      soundGunshot: /Audio/Weapons/Guns/Gunshots/laser_cannon.ogg
+    - type: CombatMode
+    - type: InteractionPopup
+      successChance: 0.5
+      interactSuccessString: petting-success-slimes
```
Comment (dmnct) [inline Resources/Prototypes/Entities/Mobs/NPCs/miscellaneous.yml:59]: This could be `petting-success-goose`

**Comment by Emisse** [comment]
expedition could use some silly mobs like this

**Comment by deltanedas** [comment]
dog

**Pull request merged**